cli/sawtooth_cli/rest_client.py

```diff
@@ -56,7 +56,11 @@ def send_batches(self, batch_list):
         Returns:
             dict: the json result data, as a dict
         """
-        data_bytes = batch_list.SerializeToString()
+        try:
+            data_bytes = batch_list.SerializeToString()
+        except AttributeError:
+            data_bytes = batch_list
+
```

Comment:
We should probably pick one interfaces then, rather than trying to implement both.